tests/unit_test/iconscore/typing/test_definition.py

```diff
@@ -142,3 +145,17 @@ def ICXTransfer(to: Address, amount: int, data: bytes):
 
     ret = _get_eventlog(ICXTransfer.__name__, signature(ICXTransfer), indexed_args_count)
     assert ret == expected
+
+
+def test__get_eventlog_without_parameter():
```

Comment:
Is double under score(`__`) typo?

Reply:
Fixed this

Reply:
There's `test__get_eventlog_without_parameter` you need to fix.